src/org/labkey/test/tests/MenuBarTest.java

```diff
@@ -211,6 +225,49 @@ public void testSteps()
         openMenu("Folders");
     }
 
+    // Issue 52171: All menus that link to absolute URLs are broken
+    @Test
+    public void testDocumentationMenuLinks()
+    {
+        goToProjectHome();
+        clickUserMenuItem(false,"LabKey Documentation");    // expect documentation to pop in another window
+        switchToWindow(1);
+        checker().withScreenshot("unexpected_destination")
+                .wrapAssertion(()-> Assertions.assertThat(getURL().toString())
+                        .as("expect navigation to labkey.org /documentation")
+                        .contains("labkey.org/Documentation"));
+        switchToMainWindow();
+        closeExtraWindows();
+
+        goToProjectHome();
+        clickUserMenuItem(true,"Support");
+        checker().withScreenshot("unexpected_destination")
+                .wrapAssertion(()-> Assertions.assertThat(getURL().toString())
+                        .as("expect navigation to localhost/home/support")
+                        .contains("localhost", "/home/support"));
```

Comment:
Dev instances aren't necessarily `localhost`.
```suggestion
        checker().withScreenshot("unexpected_destination")
                .verifyEquals("Support menu destination",
                        WebTestHelper.buildUrl("project", "home/support", "begin"),
                        getDriver().getCurrentUrl());
```